teaspoons-cli/teaspoons/commands/pipelines.py

```diff
@@ -0,0 +1,36 @@
+import click
+import typer
+
+# import generated library
+from generated.teaspoons_client import PipelinesApi
+
+# teaspoons modules
+from client import ClientWrapper
+from utils import _pretty_print
+pipelines_app = typer.Typer()
+
+
+@pipelines_app.command()
+def list():
+    try:
+        with ClientWrapper() as api_client:
+            # NOTE: this is broken until this is merged with the different api def for the get_pipelines_result
+            pipeline_client = PipelinesApi(api_client=api_client)
+            pipelines = pipeline_client.get_pipelines()
+            for pipeline in pipelines.pipelines.results:
+                click.echo(f'{pipeline.name} - {pipeline.description}')
+    except ValueError as e:
+        click.echo(str(e), err=True)
+        exit(1)
+
+@pipelines_app.command()
+@click.argument('name')
+def get_info(name: str):
+    try:
+        with ClientWrapper() as api_client:
+            pipeline_client = PipelinesApi(api_client=api_client)
+            pipeline = pipeline_client.get_pipeline_details(pipeline_name=name)
```

Comment:
just cuz im dumb. Is the idea that if we wanted to do some more fancy logic other than just make an api call we would do like `pipelines.py` -> `service/pipelines_service.py` -> `pipeline_client.py` (ignore the names i used hahaha) or something liek that where the bulk of the logic would be in that middle layer?

Reply:
yes, that's my understanding too - there's currently a missing layer for any logic that's not a straight wire-through of the thin client, since we haven't implemented any of those yet.